docker-storage-setup.sh

```diff
@@ -657,26 +657,36 @@ EOF
     Fatal "udevadm settle after partition creation failed. Exiting."
   fi
 
-  if ! wait_for_dev ${dev}1; then
-    Fatal "Partition device ${dev}1 is not available"
+  if ! wait_for_dev ${part}; then
+    Fatal "Partition device ${part} is not available"
```

Comment:
This is tricky. I suspect that after addition of udevadm settle, this code is unnecessary. Now partition should exist. So we could think of getting rid of this code entirely.